src/containers/VehiclesAddForm/VehiclesAddForm.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+import {
+  Button,
+  TextField,
+  Select,
+  MenuItem,
+  InputLabel
+} from '@material-ui/core';
+
+import './style.css';
+
+function createData(index, vehicle) {
+    return { index, vehicle };
+}
+
+const rows = [
+    createData(1, 'Corsa'),
+    createData(2, 'Uno'),
+    createData(3, 'Gol'),
+    createData(4, 'Argo'),
+    createData(5, 'Fiesta'),
+];
+
+const VehiclesAddForm = () => (
+  <form className="vehiclesAddForm" noValidate autoComplete="off">
+    <div className="vehiclesAddForm--inputs">
+        <InputLabel id="standard-basic">Marca</InputLabel>
+        <Select id="standard-basic" label="Marca">
+            {rows.map((row) => {
+                return <MenuItem value={row.index}>{row.vehicle}</MenuItem>;
+            })}
+        </Select>
```

Comment:
Por conta do `flex` na classe `vehiclesAddForm--inputs`, o elemento mãe trata esse dois elementos do Select como duas coisas separadas que podem ser alinhadas ao centro por default. E se você colocasse uma `div` em volta desse dois e colocasse `text-align: right` talvez deixaria de centralizar esses elementos.

```suggestion
      <div>
        <InputLabel id="standard-basic">Marca</InputLabel>
        <Select id="standard-basic" label="Marca">
            {rows.map((row) => {
                return <MenuItem value={row.index}>{row.vehicle}</MenuItem>;
            })}
        </Select>
      </div>
```